components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -396,6 +396,11 @@ interface KeyringServiceObserver {
   SelectedAccountChanged(CoinType coin);
 };
 
+enum OnRampProvider {
+  Wyre = 0,
```

Comment:
Enum values prefixed with 'k' are much more frequent(btw this file is not consistent in that)

Reply:
Done b3cbd80cb3dd6de16266a3015b93d87c1fca4c3a